File: launch_school/Programming_Foundations/small_problems/welcome_stranger.rb
def greetings(arr, hash)
  p "Hello, #{arr.join(' ')}! Nice to have a"\
  "#{hash[:title]} #{hash[:occupation]} around!"
end

greetings(['John', 'Q', 'Doe'], { title: 'Master', occupation: 'Plumber' })
#=> Hello, John Q Doe! Nice to have a Master Plumber around.

File: launch_school/JS210-Fundamentals-of-Javascript/lesson_five/welcome_stranger.js
function greetings(arr, obj) {
  let name = arr.join(' ');
  console.log(`Hello, ${name}! Nice to have a ${obj.title} ${obj.occupation} around.`);
}

greetings(['John', 'Q', 'Doe'], { title: 'Master', occupation: 'Plumber' });

// console output
//Hello, John Q Doe! Nice to have a Master Plumber around.
